Demo-iOS.xcodeproj/project.pbxproj

```diff
@@ -104,7 +104,6 @@
 		663FCFF616C9AF3C00CCBFB3 /* Sample Licenses from svg-android project.txt in Resources */ = {isa = PBXBuildFile; fileRef = 663FCFF516C9AF3C00CCBFB3 /* Sample Licenses from svg-android project.txt */; };
 		663FD00416CAB16B00CCBFB3 /* groups-and-layers-test.svg in Resources */ = {isa = PBXBuildFile; fileRef = 663FD00316CAB16B00CCBFB3 /* groups-and-layers-test.svg */; };
 		6649E0AD1617479200AFE92A /* QuartzCore.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6649E0AC1617479200AFE92A /* QuartzCore.framework */; };
-		6649E10D1617577000AFE92A /* libSVGKit-iOS.2.0.0.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 6649E10A1617574700AFE92A /* libSVGKit-iOS.2.0.0.a */; };
```

Comment:
Now there's no libSVGKit-iOS.x.y.z.a at all?

Reply:
That iOS Demo, using the `SVGKit.framework` for target link library. This `libSVGKit-iOS.2.0.0.a` actually does nothing than just a placeholder in the xcode project. 😅 

See this: 
![image](https://user-images.githubusercontent.com/6919743/47660570-47dbc380-dbd2-11e8-8b28-29d6ae196586.png)